backend/server.py

```
@@ -42,9 +44,23 @@
-qa_model = pipeline("question-answering")
+
+
+# Adding the LLMQuestionGenerator back so your app doesn't crash!
 llm_generator = LLMQuestionGenerator()
 
+QA_MODEL_NAME = "deepset/roberta-base-squad2"
+qa_tokenizer = AutoTokenizer.from_pretrained(QA_MODEL_NAME)
+qa_model_instance = AutoModelForQuestionAnswering.from_pretrained(QA_MODEL_NAME)
+device = 0 if torch.cuda.is_available() else -1
+
+qa_model = pipeline(
```

Ruff (0.15.6)

[error] Undefined name `pipeline`

(F821)